internal/distro/fedora/distro.go

```diff
@@ -767,6 +767,8 @@ func (t *imageType) Manifest(customizations *blueprint.Customizations,
 		mergedRepos = append(mergedRepos, repo)
 	}
 
+	mergedRepos = append(mergedRepos, customRepos...)
```

Comment:
Wasn't too sure how to work around this... this seems a bit meh. But the custom repos get removed otherwise.